### ISSUE TITLE
Class-wise DICE scores

### ISSUE THREAD

**Issue by ryankish** [cvlab-stonybrook/SAMPath#28]
Thank you for your work. Do you have the DICE scores for each class in the BCSS dataset?

**Comment by jingweizhang-xyz** [comment]
No, we only calculated the micro and macro averaged DICE. You can modify the code to make it if you want.
